File: python_collections/dictionary/word_count.py
text="hello hai hello hai hello"
words=text.split(" ") # ['hello','hai','hello','hai','hello']
dict={}
for word in words:
    if word not in dict:
        dict[word]=1
    else:
        dict[word]+=1
print(dict)
# to print the word that is highest repeating
print(max(dict,key=dict.get)) # dict.get is used to access the value directly so that highest word can printed based on the value
#sorting
print(sorted(dict,key=dict.get,reverse=True))

File: javascriptPgms/wordcount.js
var text="hello hi how hello"
var words=text.split(" ")
console.log(words);


for(let i of words)
{
    console.log(i);
}
console.log(" ");

var dict={}
for (let word of words){
if (word in dict)
{
    dict[word]+=1
}
else{
    dict[word]=1
}
}
console.log(dict);

